src/components/InsureeForm.js

```diff
@@ -31,12 +32,16 @@ const styles = (theme) => ({
 const INSUREE_INSUREE_FORM_CONTRIBUTION_KEY = "insuree.InsureeForm";
 
 class InsureeForm extends Component {
-  state = {
-    lockNew: false,
-    reset: 0,
-    insuree: this._newInsuree(),
-    newInsuree: true,
-  };
+  constructor(props) {
+    super(props);
+    this.state = {
+      lockNew: false,
+      reset: 0,
+      insuree: this._newInsuree(),
+      newInsuree: true,
+    };
+    this.isWorker = props.modulesManager.getConf("fe-insuree", "isWorker", false);
```

Comment:
Consider moving the default value to the constants. By doing so, you can ensure a single source of truth, especially since this variable is used across multiple components and its default value might vary. The best approach would be to create a DEFAULT object, which will contain this variable:
`
const DEFAULT = {
IS_WORKER: false,
};
`
And then you can use it like this:
`
this.isWorker = props.modulesManager.getConf("fe-insuree", "isWorker", DEFAULT.IS_WORKER);
`

Reply:
done